src/WP_CLI/CommandWithUpgrade.php

```diff
@@ -210,7 +210,10 @@ public function install( $args, $assoc_args ) {
 				}
 			}
 
-			if ( $result ) {
+			// Check plugin is available or not.
+			$plugin = $this->fetcher->get_many( array( $slug ) );
```

Comment:
This could just as well be a theme here, so better use the term `extension` instead:
```suggestion
			$extension = $this->fetcher->get_many( array( $slug ) );
```